style.css

```diff
@@ -0,0 +1,96 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+:root {
+  --primary-color: #6070ff;
+  --secondary-color: #7f8cff;
+  --icons-color: #505f79;
+  --headings-color: #172b4d;
+  --text-color: #344563;
+}
+
+#nav {
+  display: flex;
```

Comment:
- [ ] [ OPTIONAL ] In order to make your code DRY and more professional, I suggest using d-flex class for all flex containers in your HTML so that you are not going to repeat this code again and again in your CSS file.